### ISSUE TITLE
pcov.exclude syntax

### ISSUE THREAD

**Issue by ktamas77** [krakjoe/pcov#35]
since there are no official examples, i'd like to confirm, would this be a valid syntax to define excluded folders in `php.ini`?

```
pcov.exclude="~(vendor|tests|node_modules)~"
```

**Comment by ktamas77** [comment]
also is there a chance to specify multiple directories for the `pcov.directory` configuration item? it would be amazing if we could provide a semicolon separated list or something like that, instead of a single directory. for some legacy projects it's easier to define the 1-2 folders which needs to be covered than to exclude everything else.

**Comment by Hell4Ge** [comment]
I am wondering the same thing, I have to exclude two directories from coverage

**Comment by rask** [comment]
From the looks of it, `exclude` should accept any PCRE compliant regex, just as with the PHP PCRE (`preg_*`) functions. See https://github.com/krakjoe/pcov/blob/develop/pcov.c#L392 and https://github.com/krakjoe/pcov/blob/develop/pcov.c#L128

Edit: I mean you can use the same style of pattern matching as you would with `preg_match` in PHP.

**Comment by everton-nasc** [comment]
Hey guys, just wondering to verify if there's an option for `pcov.directory`.
I've two projects that I'd like to get the coverage, but I just got the coverage for the folder `app\`.

I tried to include another folder along with `app/` like this:
```
pcov.directory=adminProject,app
```
But it didn't work. Here's the report output:
<img width="1662" alt="Screen Shot 2020-12-11 at 5 52 02 PM" src="https://user-images.githubusercontent.com/58737540/101965323-e1f7fd80-3bd9-11eb-9fe3-06b7801ddff7.png">

Not 100% sure if we can specify multiples folders, like @ktamas77 have described.

Also, Can `pcov` recognize the filter/whitelist paths into the `phpunit.xml`?

Thanks for helping.
